app/views/shared/components/_enterprise_search.html.haml

```diff
@@ -1,6 +1,7 @@
 #active-table-search.row
   .small-12.columns
-    %input{type: :text,
+    %input{inputmode: "search",
```

Comment:
This is not very important but we can improve it: attributes should be listed by order of importance for the next reader of the code, type and model are the most important, so I think inputmode should be moved next to ofn-disable-enter as it is also a behavioral attribute.

Reply:
Thank you for the feedback Luis! I'm happy to change this.

Reply:
ok :-)
Do you want the type to be search or text?

Reply:
Very odd, my branch is showing search. I will try pushing again.

Reply:
Search is what's on GH as well, I was just asking. So search is correct.